src/main/resources/errors.yaml

```diff
@@ -350,7 +361,7 @@ request-errors:
     code: ZERO_FILTER_OPERATIONS
     title: Zero operations provided in query filter
     body: |-
-      Zero filters were provided in the filer for this query. 
+      Zero filters were provided in the filter for this query. 
```

Comment:
Title says "Zero operations provided" so this should probably also say "Zero operations" -- or alternatively `title` should be changed to refer to "filters" instead of "operations".

Also: I know `code` probably shouldn't be changed but in general "No filters provided" (or "No operations") seems better than "Zero filters provided", so maybe change that part of message too.

Reply:
Created https://github.com/stargate/data-api/issues/1662 for generally fixing error messages , will work with docs people on this